examples/data_types_time_series/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _data_types_time_series_examples:
+
+.. _data_types_time_series-examples-index:
+
+Data Types: Time Series
```

Comment:
I think this would probably just be better as `'TimeSeries'` or `'Time Series Data'`.

Which makes me wonder, would it be better to have these titles not be the names of the classes but a description of the type of data? I.e 'Time Series Data' and '2D Images' for this and maps?

Reply:
I think it makes sense to follow the documentation we've written before -- so people can just find examples of what they've already read about. In the [Brief Tour of SunPy](http://docs.sunpy.org/en/stable/guide/tour.html), we talk about Maps, Time Series, and Spectra, so I followed that convention.